vividus/src/main/java/org/vividus/results/ResultsProvider.java

```diff
@@ -17,19 +17,14 @@
 package org.vividus.results;
 
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 
 import org.vividus.report.allure.model.Status;
 import org.vividus.results.model.Failure;
-import org.vividus.results.model.NodeType;
-import org.vividus.results.model.Statistic;
 
-public interface ResultsProvider
+public interface ResultsProvider extends StatisticsProvider
```

Comment:
just move full `ResultsProvider` to `vividus-engine`

Reply:
ResultsProvider contains Status entity from allure so I just get those things that I will use

Reply:
addressed in separate PR